webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/scripts.js',
+  },
```

Comment:
You have specified the entry file as `./src/scripts.js`, however, your script file is named `./src/script.js`. Kindly modify this to match so as to avoid errors.

Reply:
Hello @emmiiorji, Thank you for you comment and I will fix it.